cli/tests/unit/console_test.ts

```diff
@@ -392,14 +392,14 @@ Deno.test(function consoleTestStringifyFunctionWithProperties() {
   assertEquals(
     stringify({ f }),
     `{
-  f: [Function: f] {
+  f: [Function: f] <ref *1> {
```

Comment:
I am not happy with this. In node the ref is before `[Function: f]` or any other description.
This is also inconsistent behaviour as
```js
class Foo {}
const x = new Foo();
x.y = x;
console.log(x);
```
Will output `<ref *1> Foo { y: [Circular *1] }`

Reply:
Can't `<ref *1>` be prepended in this case?

Reply:
Maybe let's make `inspectRawObject` returning `[inspected, refIndex]` array, and prepend `<ref *1>` part inside of `inspectFunction`? (around here: https://github.com/denoland/deno/blob/26f5c223df26962f19ccc957d0507facc09c55c5/ext/console/02_console.js#L358-L362